src/api/app/views/webui/main/_status_messages.html.haml

```diff
@@ -10,7 +10,6 @@
     - if User.admin_session?
       - unless flipper_responsive?
         .card-footer
-          = link_to('#', 'data-toggle': 'modal', 'data-target': '#status-message-modal') do
+          = link_to(new_status_message_path, class: 'nav-link') do
             %i.fas.fa-plus-circle.text-primary
-            Add status message
-      = render partial: 'add_status_message_modal'
+            Add Status Message
```

Comment:
To have consistent naming in links, let's use `Create Status Message`

Reply:
done!